tests/integration/targets/setup_view/tasks/main.yml

```
@@ -0,0 +1,16 @@
+---
+- module_defaults:
+    group/infoblox.bloxone.all:
+      csp_url: "{{ csp_url }}"
+      api_key: "{{ api_key }}"
+  block:
+    # Create a random View name to avoid conflicts
+    - ansible.builtin.set_fact:
+        name: "test-view-{{ 999999 | random | string }}"
+
+    - name: Create a View
+      infoblox.bloxone.dns_view:
+        name: "{{ name }}"
+        state: present
+      check_mode: true
```

```suggestion
      check_mode: true
```

a check_mode would mean it is a dry run and the view is not actually created. 

All the tests above don't use the view parameter, which means they are being created in the default view and not this view.

---

Fixed